Mapped lots of random things

Shnupbups opened the pull request:
Mostly function arguments, but plenty of other things too. Fixed some mistakes I found also.

Shnupbups committed:
Stuff

Shnupbups committed:
more stuff

Shnupbups committed:
even more stuff

Shnupbups committed:
removed an accidental thing

Shnupbups committed:
stuff 4: electric boogaloo

asiekierka commented on mappings/net/minecraft/block/BlockState.mapping:
```diff
@@ -6,6 +6,7 @@ CLASS net/minecraft/class_2680 net/minecraft/block/BlockState
 		FIELD field_16558 opaque Z
 		FIELD field_16559 DIRECTIONS [Lnet/minecraft/class_2350;
 		FIELD field_16560 shapes [Lnet/minecraft/class_265;
+		FIELD field_19429 isSolidFullSquare [Z
```
solidFullSquare

Shnupbups replied:
Good point, will change.

asiekierka commented on mappings/net/minecraft/block/BlockState.mapping:
```diff
@@ -200,3 +201,4 @@ CLASS net/minecraft/class_2680 net/minecraft/block/BlockState
 		ARG 2 state
 		ARG 3 hitResult
 		ARG 4 projectile
+	METHOD method_20827 isSolidFullSquare (Lnet/minecraft/class_1922;Lnet/minecraft/class_2338;Lnet/minecraft/class_2350;)Z
```
How can a 3D model be a square? Or is this side-specific, in which case the method should probably say so?

Shnupbups replied:
It's for checking if a specific side of a blockstate is a solid full square. It uses the same name as the method it calls from `Block`.

asiekierka replied:
isSideSolidFullSquare, then

Shnupbups replied:
Well then should the method in `Block` not also be `isSideSolidFullSquare`?

asiekierka replied:
Correct

Shnupbups replied:
![image](https://user-images.githubusercontent.com/2484721/61578801-6fe3cc00-ab3f-11e9-8485-ea4717e99230.png)
This is the code, the method in `Block` is also side specific and is named `isSolidFullSquare`. I named the `BlockState` method the same for consistency. I can change both, but I don't see why one should be different to the other.

Shnupbups replied:
Done

asiekierka commented on mappings/net/minecraft/client/input/KeyCombo.mapping:
```diff
@@ -0,0 +1,7 @@
+CLASS net/minecraft/class_4324 net/minecraft/client/input/KeyCombo
```
KeyCombo? Not KeyCombination? It's also made out of chars...

Shnupbups replied:
It's a Mojang name. See the class' `toString`.

asiekierka commented on mappings/net/minecraft/client/util/math/PosAndRot.mapping:
```diff
@@ -0,0 +1,11 @@
+CLASS net/minecraft/class_4462 net/minecraft/client/util/math/PosAndRot
```
That's a bad name

Shnupbups replied:
It's a Mojang name. See the class' `toString`.

asiekierka commented on mappings/net/minecraft/realms/RealmsError.mapping:
```diff
@@ -0,0 +1,8 @@
+CLASS net/minecraft/class_4345 net/minecraft/realms/RealmsError
```
Are these not mapped by Rune's PR?

Shnupbups replied:
Maybe. In all honesty I haven't checked.

asiekierka replied:
Time to check!

Shnupbups replied:
It is, as are a few other classes. I have removed them from this PR.

Shnupbups commented:
They may be mapped by Rune's PR, I didn't see it before making this one.

As for the names of PosAndRot and KeyCombo, they are the Mojang names. (as can be seen by their toString methods)

And as for `isSolidFullSquare` it is for checking whether a specific side of a block state is a full square. It's named the same as the method from `Block` that it calls.

Shnupbups committed:
Update BlockState.mapping

Shnupbups committed:
Stuff VI: The Search for Stuff V

Shnupbups committed:
Update NativeImage.mapping

Shnupbups committed:
removed things already mapped by Rune's PR

Namely `net/minecraft/realms/YesNoScreen' (`com/mojang/realmsclient/gui/screens/RealmsConfirmScreen` in Rune's PR), `net/minecraft/realms/RealmsError' (`com/mojang/realmsclient/client/RealmsError` in Rune's PR), and `net/minecraft/realms/InvalidClientScreen' (`com/mojang/realmsclient/gui/screens/RealmsClientOutdatedScreen` in Rune's PR).

Shnupbups committed:
Stuff: The Final Stuffening

Shnupbups committed:
Merge pull request #1 from FabricMC/1.14.4-pre6

1.14.4 pre6

Shnupbups commented:
New version at #794
